### PR TITLE
dgst enablement

### LINKED ISSUE #51: Investigate use of dgst command
Plain QSC algs work, hybrids have failures. Reason unknown.

### PR DESCRIPTION
**Pull request by baentsch**
Fixes #51

### CODE REVIEW

File: oqsprov/oqsprov_keys.c [hunk 1]
```diff
@@ -830,7 +830,7 @@ int oqsx_key_maxsize(OQSX_KEY *key) {
     case KEY_TYPE_SIG:
 	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature;
     case KEY_TYPE_HYB_SIG:
-	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature + key->oqsx_provider_ctx.oqsx_evp_ctx->evp_info->length_signature;
+	return key->oqsx_provider_ctx.oqsx_qs_ctx.sig->length_signature + key->oqsx_provider_ctx.oqsx_evp_ctx->evp_info->length_signature+SIZE_OF_UINT32;
```
Comment (dstebila) [inline oqsprov/oqsprov_keys.c:833]: Does this reflect a bug in the earlier code?

File: README.md [hunk 2]
```diff
@@ -202,12 +202,31 @@ Continuing the example above, the following command verifies the CMS file
...
+
+#### Verifying
+
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl dgst -provider-path _build/oqsprov -provider oqsprovider -provider default -signature dgstsignfile -verify qsc.pubkey inputfile
+
+The public key can be extracted from the certificate using standard openssl command:
+
+    LD_LIBRARY_PATH=.local/lib64 .local/bin/openssl x509 -provider-path _build/oqsprov -provider oqsprovider -provider default -in qsc.crt -pubkey -noout > qsc.pubkey
+
+The `dgst` command is not tested for interoperability with [oqs-openssl111](https://github.com/open-quantum-safe/openssl).
```
Comment (dstebila) [inline README.md:228]: Re: "not tested for interoperability": Is this just a disclaimer that we haven't tested it? Or indicating that it's actually known to be not interoperable?  The former is fine, but the latter would be less good.

File: oqsprov/oqsprov_keys.c (hunk 1 above)
Comment (baentsch) [reply oqsprov/oqsprov_keys.c:833]: "Reflect". Nice term. Yes: The previous hybrid sig code was buggy, indeed.

File: README.md (hunk 2 above)
Comment (baentsch) [reply README.md:228]: It only means to say it's not tested. Someone mentioned that oqs-openssl has a problem with `dgst` without ever giving a concrete sample (https://github.com/open-quantum-safe/openssl/issues/357) and I didn't try to set up interop testing. Do you deem it sensible/necessary?

**Review by dstebila — APPROVED** [review]

**Pull request merged**